SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorCommandLineArgsManager.cpp

```diff
@@ -0,0 +1,304 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialGDKEditorCommandLineArgsManager.h"
+
+#include "Input/Reply.h"
+#include "IOSRuntimeSettings.h"
+#include "Logging/LogMacros.h"
+#include "Misc/FileHelper.h"
+#include "Misc/MessageDialog.h"
+#include "Serialization/JsonSerializer.h"
+#include "SpatialCommandUtils.h"
+#include "SpatialGDKEditorSettings.h"
+#include "SpatialGDKSettings.h"
+
+#ifdef ENABLE_LAUNCHER_DELEGATE
+//need this macro before "ILauncherServicesModule.h" to prevent compile error
+#define LAUNCHERSERVICES_API
+
+#include "Developer/LauncherServices/Public/ILauncherServicesModule.h"
+#include "Developer/LauncherServices/Public/ILauncherWorker.h"
+#include "Developer/LauncherServices/Public/ILauncher.h"
```

Comment:
Add "LauncherServices" to `PrivateDependencyModuleNames` in `SpatialGDKEditor.build.cs` so the module is properly included.
```suggestion
#include "ILauncherServicesModule.h"
#include "ILauncherWorker.h"
#include "ILauncher.h"
```